lib/ddtrace/ext/distributed.rb

```diff
@@ -6,6 +6,7 @@ module DistributedTracing
       HTTP_HEADER_TRACE_ID = 'x-datadog-trace-id'.freeze
       HTTP_HEADER_PARENT_ID = 'x-datadog-parent-id'.freeze
       HTTP_HEADER_SAMPLING_PRIORITY = 'x-datadog-sampling-priority'.freeze
+      SAMPLING_PRIORITY_KEY = '_sampling_priority_v1'.freeze
```

Comment:
I'm not sure if there is a better place for this constant. Let me know your thoughts.

Reply:
here is fine; when moving to OpenTracing we'll use directly their constants.

Reply:
I'm fine with it being here, also not totally happy, but think it's OK.

Reply:
yes we can simply use OpenTracing tags later on, when our clients have a proper migration.